index.js

```diff
@@ -1,93 +1,53 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import bodyParser from 'body-parser';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerui from 'swagger-ui-express';
+//import swaggerDoc from './docs/swagger';
 
-const isProduction = process.env.NODE_ENV === "production";
 
 // Create global app object
 const app = express();
 
-app.use(cors());
+// swagger definition
+const swaggerDefinition = {
+    info: {
+      title: 'Authors Haven',
+      version: '1.0.0',
+      description: 'A Social platform for the creative at heart',
+    },
+    host: '',
```

Comment:
You need to make the host field configurable by using environment variables. You can do this by using [dotenv](https://www.npmjs.com/package/dotenv) package.
```suggestion
    host: `${process.env.BASE_URL}`,
```